### PR TITLE
Show JMX username in ui

### PR DESCRIPTION
**Pull request by ossarga**

### CODE REVIEW

File: src/ui/app/jsx/cluster-list.jsx [hunk 1]
```diff
@@ -104,7 +106,11 @@ const Cluster = React.createClass({
                                    style={progressStyle} 
                                    label={repairRun.keyspace_name}
                                    key={repairRun.id}/>
-      )
+      );
+
+      if (repairProgress === undefined || repairProgress.length == 0) {
+          repairProgress = (<ProgressBar bsStyle="success" now={0} label={"No_repairs"}/>);
+      }
```
Comment (michaelsembwever) [inline src/ui/app/jsx/cluster-list.jsx]: what's this? 🤷‍♂️

**Review by michaelsembwever — APPROVED** [review]
LGTM.

There's some stuff in the UI added around running repairs on the cluster. Not sure where that came from.

File: src/ui/app/jsx/cluster-list.jsx (hunk 1 above)
Comment (ossarga) [reply src/ui/app/jsx/cluster-list.jsx]: This change adds an empty status bar for the case where there are no active repairs.

Comment (ossarga) [reply src/ui/app/jsx/cluster-list.jsx]: As discussed we can drop this change and put in a separate PR.

**Review by michaelsembwever — APPROVED** [review]
:shipit:

**Comment by ossarga** [comment]
Skipping over the 4.0 tests as the CCM link to clone trunk is broken.

**Pull request merged**